lib/gui/app.js

```diff
@@ -160,23 +160,25 @@ app.run(() => {
   store.subscribe(() => {
     const currentFlashState = flashState.getFlashState();
 
+    if (!flashState.isFlashing()) {
+      return;
+    }
+
     // There is usually a short time period between the `isFlashing()`
     // property being set, and the flashing actually starting, which
     // might cause some non-sense flashing state logs including
     // `undefined` values.
     //
     // We use the presence of `.eta` to determine that the actual
     // writing started.
```

Comment:
@Shou Is `(flashState.isFlashing() && !currentFlashState.eta)` a reliable indicator of being in the 'elevating' stage? (or does `flashState.isFlashing()` only get set post-elevation?)